app/views/episode_transcripts/_form.html.erb

```diff
@@ -5,19 +5,24 @@
   <%= form.submit class: "d-none" %>
 
   <%= turbo_frame_tag "episode-form-transcript", data: {controller: "media-upload"} do %>
-    <div class="container">
-      <div class="row mt-4 mx-2">
-        <div class="col-lg-8">
-          <div class="row">
-            <%= render "form_transcript", episode: episode, form: form %>
-            <%= render "form_main", episode: episode, form: form %>
+    <% episode.build_transcript if episode.transcript.blank? %>
+    <% episode.transcript.clear_attribute_changes(%i[format]) %>
```

Comment:
i did a little bit of rearranging with the partials; technically not needed in this PR but it feels like both `form_transcript` and `form_main` both needed `form.fields_for :transcript` anyway?

i did this because i originally thought i was going to use `fields.updated_at` in `form_status`, but i opted to be more direct with `episode.transcript.updated_at`

Reply:
+1 makes sense.  And I'm all for reorganizing!